lametro/templates/lametro/index.html

```diff
@@ -136,5 +136,34 @@ <h2>
             collapseActivity();
             return false;
         });
+
+        $('#search-bar').autocomplete({
+          serviceUrl: '{% url "autocomplete" %}',
+          paramName: 'query',
+          transformResult: function(response) {
+            var response = JSON.parse(response);
+            console.log(response);
+            return {
+              suggestions: $.map(response.termHints, function(d) {
+                return {'value': d.name, 'data': d.id};
+              })
+            }
+          },
+          onSelect: function(suggestion) {
+            $.when(
+              $.get('/topic/', { 'guid': suggestion.data }))
+                .then(function(response) {
+                  if (response.status_code == 200) {
+                    var base = '/search/?selected_facets=topics_exact%3A';
+                    var url = base + response.subject_safe;
+                    window.location.href = url;
+                  } else {
+                    console.log(response)
```

Comment:
@hancush what should the exception behavior be here you think?

Reply:
we don't have great patterns for logging in a useful way in javascript. we have sentry configured to capture exceptions in our routes, so i think don't do anything if we don't have a 200 status code, for now.